src/openzaak/components/catalogi/tests/admin/test_admin_publish.py

```diff
@@ -316,6 +317,99 @@ def test_publish_zaaktype_related_to_concept_informatieobjecttype_fails(self, m)
         self.assertIsNone(publish_button)
 
 
+@requests_mock.Mocker()
+class PublishWithGeldigheidTests(
```

Comment:
Nice test! And thanks for catching that if we relax overlapping check for creating concept resources, we need to add one when we publish them

Could I also ask you to add the similar test for API? I'm afraid that now we can publish overlapping objects via API.